template/extra_sprig.go

```diff
@@ -7,25 +7,25 @@ import (
 )
 
 const (
-	sprigCrypto         = "Sprig Cryptographic & Security, http://masterminds.github.io/sprig/crypto.html"
-	sprigDate           = "Sprig Date, http://masterminds.github.io/sprig/date.html"
-	sprigDict           = "Sprig Dictionnary, http://masterminds.github.io/sprig/dicst.html"
-	sprigEncoding       = "Sprig Encoding, http://masterminds.github.io/sprig/encoding.html"
-	sprigFilePath       = "Sprig File Path, http://masterminds.github.io/sprig/paths.html"
-	sprigFlow           = "Sprig Flow Control, http://masterminds.github.io/sprig/flow_control.html"
-	sprigGen            = "Sprig General, http://masterminds.github.io/sprig/"
-	sprigList           = "Sprig List, http://masterminds.github.io/sprig/lists.html"
-	sprigMath           = "Sprig Mathematics, http://masterminds.github.io/sprig/math.html"
-	sprigRegex          = "Sprig Regex, http://masterminds.github.io/sprig/strings.html"
-	sprigSemver         = "Sprig Version comparison, http://masterminds.github.io/sprig/semver.html"
-	sprigString         = "Sprig Strings, http://masterminds.github.io/sprig/strings.html"
-	sprigReflect        = "Sprig Reflection, http://masterminds.github.io/sprig/reflection.html"
-	sprigDefault        = "Sprig Default, http://masterminds.github.io/sprig/defaults.html"
-	sprigOS             = "Sprig OS, http://masterminds.github.io/sprig/defaults.html"
-	sprigTypeConversion = "Sprig Type Conversion, http://masterminds.github.io/sprig/conversion.html"
-	sprigStringList     = "Sprig String Slice, http://masterminds.github.io/sprig/string_slice.html"
-	sprigURL            = "Sprig URL functions, http://masterminds.github.io/sprig/url.html"
-	sprigNetwork        = "Sprig Network functions, http://masterminds.github.io/sprig/network.html"
+	sprigCrypto         = "Sprig Cryptographic & Security"
+	sprigDate           = "Sprig Date"
+	sprigDict           = "Sprig Dictionary"
+	sprigEncoding       = "Sprig Encoding"
+	sprigFilePath       = "Sprig File Path"
+	sprigFlow           = "Sprig Flow Control"
+	sprigGen            = "Sprig General"
+	sprigList           = "Sprig List"
+	sprigMath           = "Sprig Mathematics"
+	sprigRegex          = "Sprig Regex"
+	sprigSemver         = "Sprig Version comparison"
+	sprigString         = "Sprig Strings"
+	sprigReflect        = "Sprig Reflection"
+	sprigDefault        = "Sprig Default"
+	sprigOS             = "Sprig OS"
+	sprigTypeConversion = "Sprig Type Conversion"
+	sprigStringList     = "Sprig String Slice"
+	sprigURL            = "Sprig URL functions"
+	sprigNetwork        = "Sprig Network functions"
```

Comment:
Why do you remove the links?

Reply:
Links in menus or headings don't work well. A lot of them were wrong too

Reply:
Initially, I was using them in the online doc. They were all working then 😁. Not sure that removing them was the best solution.

Reply:
hmm. I will split on a comma and put the first part as the title and the second part under it. That way, we'll have title and link in the doc